src/passwordless/client.py

```diff
@@ -152,6 +154,17 @@ def delete_user(self, delete_user: DeleteUser) -> None:
         """
         pass
 
+    @abstractmethod
+    def send_magic_link(self, send_magic_link: SendMagicLinkRequest) -> None:
+        """Sends a magic link.
+
+        :param send_magic_link: `DeleteUser` containing details about the
```

Comment:
typo, should be `SendMagicLinkRequest`

Reply:
saw it got fixed in https://github.com/bitwarden/passwordless-python/pull/24/files